src/main/java/com/passbook/sparkeighteen/controller/UserController.java

```diff
@@ -38,4 +42,10 @@ public ResponseEntity<LoginResponse> login(@Valid @RequestBody final LoginReques
         return new ResponseEntity<>(userService.login(request), HttpStatus.OK);
     }
 
+    @ApiOperation("Update user profile")
+    @PutMapping("/user/{userID}/profile")
+    public ResponseEntity<ProfileResponse> updateProfile(@PathVariable Integer userID, @Valid @RequestBody final ProfileRequest request) throws Exception {
+        return new ResponseEntity<>(userService.updateProfile(userID, request),HttpStatus.OK);
```

Comment:
There will be issue in validation did you guys verified?

Reply:
Yes sir It's working.